packages/app-project/stores/Project.js

```diff
@@ -0,0 +1,44 @@
+import { flow, types } from 'mobx-state-tree'
+import _ from 'lodash'
+import axios from 'axios'
+
+function transformResponse (response) {
+  return _.get(response, 'data.projects[0]')
+}
+
+function fetchProjectData (slug) {
+  return axios({
+    url: 'https://www.zooniverse.org/api/projects',
+    params: { slug },
+    headers: {
+      'Accept': 'application/vnd.api+json; version=1',
+      'Content-Type': 'application/json'
+    }
+  }).then(transformResponse)
+}
+
+const Project = types
+  .model('Project', {
+    isLoading: types.boolean,
```

Comment:
Instead of a single `isLoading` boolean, would you consider a more explicit approach to the state of the data loading? Here's an example of what I mean from the edu-api front-end and a pattern @shaunanoordin suggested: https://github.com/zooniverse/edu-api-front-end/blob/master/src/ducks/classrooms.js#L6-L16

We have explicit constants for all kinds of requests and outcomes. It's set throughout the async action. Here's the request for a classroom as an example: https://github.com/zooniverse/edu-api-front-end/blob/master/src/ducks/classrooms.js#L30

Reply:
Absolutely - I'll try adding this in